packages/grant-explorer/src/features/discovery/LandingPage.tsx

```diff
@@ -13,6 +13,10 @@ import ApplyNowSection from "./ApplyNowSection";
 const LandingBannerLogo = lazy(() => import("../../assets/LandingBanner"));
 
 const LandingPage = () => {
+  if (process.env.REACT_APP_ENV === "production") {
```

Comment:
It will work but the correct version would be:

```ts
useEffect(() => { 
  // ...
}, [])
```

Reply:
@bard you are right, I just re-enabled the old one but I'll write it in the correct way